sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```
@@ -1445,6 +1447,10 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog with QueryErrorsB
             && SQLConf.get.getConf(SQLConf.DECORRELATE_SET_OPS_ENABLED))
           p.children.foreach(child => checkPlan(child, aggregated, childCanContainOuter))
 
+        // SQLFunctionNode serves as a container for the underlying SQL function plan.
+        case s: SQLFunctionNode =>
```

how can we hit it? I think we will throw UNSUPPORTED_SQL_UDF_USAGE earlier.